test/integration/tags/echo_test.rb

```diff
@@ -4,8 +4,8 @@ class EchoTest < Minitest::Test
   include Liquid
 
   def test_echo_outputs_its_input
-    assert_template_result('BAR', <<~LIQUID, { 'variable-name' => 'bar' })
-      {%- echo variable-name | upcase -%}
-    LIQUID
+    assert_template_result('BAR', <<-LIQUID, { 'variable-name' => 'bar' })
+{%- echo variable-name | upcase -%}
```

Comment:
this indentation kinda sucks 😢

Reply:
The alternative to this approach is bumping Ruby to 2.3

Reply:
What does that have to do with those new tags though?

Reply:
`<<~HEREDOC` is only support in 2.3+

Reply:
I'd rather not use heredoc at all here if it forces us to use this ugly indentation style

Reply:
Currently liquid is `2.1+`

Reply:
I can use https://github.com/JuanitoFatas/String-strip_heredoc-patch/blob/master/strip_heredoc.rb on this to make it pretty